sdk/servicebus/service-bus/src/models.ts

```diff
@@ -105,8 +105,8 @@ export interface ServiceBusReceiverOptions {
 
   /**
    * The maximum duration in milliseconds until which the lock on the message will be renewed
-   * by the sdk automatically. This auto renewal stops once the message is settled or once the user
-   * provided onMessage handler completes ite execution.
+   * by the sdk automatically. This auto renewal stops once the message is settled(or once the user
+   * provided processMessage handler completes its execution if used `ServiceBusReceiver.subscribe()` method).
```

Comment:
This option does not get used when creating a session receiver. This is because the `acceptSession` and `acceptNextSession` methods take `ServiceBusSessionReceiverOptions and not `ServiceBusRecieverOptions`. So, we should not need this change

Reply:
No, I meant to differentiate it for the `receiveMessages` since onMessage is not applicable for that.

Reply:
This option is applicable for messages received in both `receiveMessages` and `subscribe`, but the docs for this option mention `onMessage` as if it is used only for `subscribe`.

Reply:
Hmmm... then that is an in-consistent story we will be telling... and will be confusing...

I would expect the auto lock renewal to behave the same way regardless of which method is used to receive messages. It should stop if the timer runs out or if the message is settled. 

In case of subscribe(), if auto complete is enabled, then message is settled after callback is executed. If user callback fails, then message is abandoned. But if auto complete is disabled and user callback does not throw error, why should lock renewal stop?

Reply:
Good point, I agree to be consistent in both the places, lock renewal should be independent of the processMessage handler. 
Should we update it before GA?

Reply:
Well... am not 100% convinced of this, lets discuss tomorrow

Reply:
Tested this case, I can see the lock being renewed after my `processMessage` has ended. So, I'm removing the processMessage part in the doc comment.